packs/aws/requirements.txt

```diff
@@ -1 +1,2 @@
 boto
+boto3
```

Comment:
I already had this code written for boto3 rewriting it for boto would be more time consuming.